src/main/java/org/levelup/labradoodle/controllers/HomeController.java

```diff
@@ -63,7 +63,12 @@ public DishesDto getDishById(@PathVariable(value = "id") int id) {
     @ResponseBody
     @RequestMapping(value = "/get_dishes_type", method = RequestMethod.GET)
     public ArrayList<TypeDishes> getDishesType() {
-        return new ArrayList<TypeDishes>
-                (TypeDishes.PIZZA, TypeDishes.CHINE, TypeDishes.BURGERS, TypeDishes.FISH, TypeDishes.SUSHI);
+        ArrayList dishesType = new ArrayList<TypeDishes>();
```

Comment:
1. List should be filled with values not in controller.. Do it in service layer, and then call this service method from controller
2. ArrayList dishesType = new ArrayList<TypeDishes>();      - Why u not select list generic type near field? Its object now
3. dishesType.add("PIZZA");
Why u adding strings? add enum values
4. Why u add manualy every value? use 
```
TypeDishes.values()
```
to get array of all values, and than  add then in 'for' to the list... that'd  allow  you to not add manually enum values every time when TypeDishes got new type